TranslationBackend/routes/api.php

```diff
@@ -22,6 +22,5 @@
 //route is prepended with /api/ prefix
 Route::get('/translations', 'WordController@show');
 
-// route below is only used to set up the database, this shouldn't be exposed to the public
-//Route::get('/import', 'ImportController@store');
-
+Route::get('/uploadfile','UploadDataController@index');
```

Comment:
do we need this route? When do we use it?

Reply:
I'll comment it out when we deploy it